README.md

```diff
@@ -10,10 +10,34 @@ It's totally not the first approach of this kind. But: this one should be easily
 
 Currently, Wireshell  consists of the following basic commands:
 
-#### New
+#### Fresh installation
```

Comment:
I have changed, may be you make sure the thing I did is what you asked.

Thank you